storm-client/src/jvm/org/apache/storm/daemon/worker/WorkerState.java

```diff
@@ -414,18 +408,16 @@ public void refreshConnections() {
         cachedNodeToPortSocket.getAndUpdate(prev -> {
             Map<NodeInfo, IConnection> next = new HashMap<>(prev);
             for (NodeInfo nodeInfo : newConnections) {
-                next.put(nodeInfo,
-                         mqContext.connect(
-                             topologyId,
-                             //nodeHost is not null here, as newConnections is only non-empty if assignment was not null above.
-                             nodeHost.get(nodeInfo.get_node()),    // Host
-                             nodeInfo.get_port().iterator().next().intValue(),       // Port
-                             workerTransfer.getRemoteBackPressureStatus()));
+                next.put(nodeInfo, mqContext.connect(topologyId,
```

Comment:
Please either move mqContext.connect back on its own line, or make a new variable so we can separate the connect and put calls. It's not obvious at a glance now which call the parameters below belongs to.

Reply:
Looks like i accidentally hit the auto-style shortcut at some point. I'll revert all the style changes that aren't part of the changes.